dbt/adapters/snowflake/impl.py

```diff
@@ -135,7 +135,8 @@ def list_relations_without_caching(self, schema_relation: SnowflakeRelation) ->
             # if the schema doesn't exist, we just want to return.
             # Alternatively, we could query the list of schemas before we start
             # and skip listing the missing ones, which sounds expensive.
-            if "Object does not exist" in str(exc):
+            # The error messages are different in English and Japanese, So need each handling.
+            if "Object does not exist" in str(exc) or "オブジェクトは存在しない" in str(exc):
```

Comment:
alternatively perhaps the error code itself is worth catching? `002043 (02000)`
```suggestion
            # "002043 (02000)" is error code for "object does not exist or is not found"
            # The error messages are different in English and Japanese, need each handling.
            if "002043 (02000)" in str(exc):
```

Reply:
This is best.